_targets.R

```diff
@@ -13,15 +13,19 @@ options(tidyverse.quiet = TRUE)
 
 source('1_fetch.R')
 source('2_process.R')
-# source('3_visualize.R')
+source('3_visualize.R')
 
 # tar_option_set(debug = "p1_pest_bin_csv")
-# year <- '2019'
+
 
 # define input file paths
-path_dbfs <- c('1_fetch/in/maps/dbf19')
-path_bins <- c('1_fetch/in/maps/bins')
-path_labels <- c('1_fetch/in/maps/labels')
+path_base <- c('1_fetch/in/maps/')
+path_dbfs <- paste0(path_base, 'dbf19')
+path_bins <- paste0(path_base, 'bins')
+path_labels <- paste0(path_base, 'labels')
+
+# define plot variables
+year <- '2019'
```

Comment:
I'm not sure what the best practices are for what is created in vs out of the pipeline like this, but it makes sense to me to have these things here like a congif file since they may be the primary elements of the pipeline that change.

Reply:
Yes, I did it this way because any one of these subfolder names could change from year to year.